### DISCUSSION TITLE
Matterbridge crashes at start

### DISCUSSION THREAD

**Discussion by fastcrafteYT-dev** [42wim/matterbridge#1632] [Help / Q&A]
Hello, I get an error when I start Matterbridge and it says that I can't connect to anything, this happened from one day to the next. I have tried reinstalling it several times. 

I hope it is possible to help me.

**Comment by Mikaela** [comment]
What is the crashing error message and what does your config file look like? Which version and OS and architecture are you using?

**Reply by fastcrafteYT-dev** [reply]
Hello, 
I use Debian 10 and the version 1.22.3 for linux 64-bit. 

https://i.imgur.com/mX7UkWo.png
This is the link to the error in the console https://i.imgur.com/WTIfDV3.png
